uport-did/src/main/java/me/uport/sdk/uportdid/UportIdentityDocument.kt

```diff
@@ -113,4 +113,22 @@ data class UportDIDDocument(
         override val context: String = "https://w3id.org/did/v1",
 
         val uportProfile: UportIdentityDocument
-) : DIDDocument
\ No newline at end of file
+) : DIDDocument {
```

Comment:
We should keep the naming consistent
`UportIdentityDocument` -> `UportDIDDocument`

Reply:
I don't understand the inconsistency.
The `UportIdentityDocument` is the legacy model for the profile document, that didn't conform to DID standards.
`UportDIDDocument` conforms to the standards and can be obtained from the legacy model.

I'll move it to a different file and apply some kdoc to clear up misunderstandings